cmd/cosign/cli/fulcio/fulcio.go

```diff
@@ -53,11 +59,26 @@ const (
 	altRoot    = "SIGSTORE_ROOT_FILE"
 )
 
+type Resp struct {
+	CertPEM  []byte
+	ChainPEM []byte
+	SCT      []byte
+}
+
 // This is the root in the fulcio project.
 //go:embed fulcio.pem
 var rootPem string
+
+// This is the public key for the fulcio CT log
+//go:embed ct.pub
+var ctPublicKey string
```

Comment:
this is also found in the sigstore TUF root under `ctfe.pub`
similar to https://github.com/asraa/cosign/blob/4b3ec76463cdfa1dd9cb94fbc03ae11c07be5c07/cmd/cosign/cli/fulcio/fulcio.go#L214

Reply:
thanks!